types/properties-reader/index.d.ts

```diff
@@ -18,6 +18,9 @@ declare namespace PropertiesReader {
         each<T>(iterator: (this: T, key: string, value: Value) => void, scope: T): Reader;
         getAllProperties(): { [key: string]: Value; };
         clone(): Reader;
+        save(destFile: string): Promise<any>;
+        getByRoot(root: any): {};
```

Comment:
As I understand the source, this should return `{ [key: string]: value }`.

Reply:
Also fixed